decidim-proposals/lib/decidim/proposals/engine.rb

```diff
@@ -88,6 +94,14 @@ class Engine < ::Rails::Engine
         end
       end
 
+      initializer "decidim_proposals_mentioned_listener" do
+        Decidim::Comments::CommentCreation.subscribe do |data|
```

Comment:
I don't agree because there may be the case that other Decidim components will be interested in CommentCreation (user mentions, tags, linking of other components like meetings, etc..)
Don't you think that subscribing to this event has high probabilities of being reused?